tests/tfm/tfm_psa_test/testcase.yaml

```diff
@@ -0,0 +1,23 @@
+common:
+  tags: tfm
+  platform_allow: nrf5340dk_nrf5340_cpuapp_ns nrf9160dk_nrf9160_ns
```

Comment:
Please add integration platform option to specify which exactly platforms should be used in Jenkins CI and to help us avoid "silent skips" which are possible when we rely only on `platform_allow` option:
```yaml
integration_platforms:
  - nrf5340dk_nrf5340_cpuapp_ns
  - nrf9160dk_nrf9160_ns
```